### ISSUE TITLE
Create a guide for updating and adding features to the project.

### ISSUE THREAD

**Issue by shaleenanuj** [rohitkolapkar/Pre-Reconstruction-Image-Processing#1]

**Comment by rohitkolapkar** [comment]
Yes sir, I will add a guide in my repository as soon as possible. 
Till then to use this plugin in imageJ, download the jar file from [here](https://github.com/rohitkolapkar/Pre-Reconstruction-Image-Processing/blob/master/IJ/plugins/Pre_Reconstruction_Processes_.jar) and add it to the plugins directory of ImageJ in your desktop.
